File: ConsoleApp1/HomeWork17.04.25/Prime.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HomeWork17._04._25
{
    internal class Prime
    {
        static void Main(string[] args)
        {
            int a = Convert.ToInt32(Console.ReadLine());
            if (a <= 1) Console.WriteLine("Ko là snt");
            else
            {
                bool IsPrime = true;
                for (int i = 2; i <= Math.Sqrt(a); i++)
                {
                    if (a % i == 0) IsPrime = false;
                }
                Console.WriteLine(IsPrime ? "SNT" : "Không SNT");
            }

        }
    }
}

File: Prime.py
import math
n = input()
n = int(n)
if(n <= 1): 
    print("Khong phai so nguyen to")
else:
    IsPrime = True
    for i in range(2, math.sqrt(n) + 1, 1):
        if(n % i == 0): IsPrime = False
    print(f"{n} SNT" if IsPrime else f"{n} Khong phai SNT")
    # if (IsPrime): print(f"{n} la so nguyen to")
    # else: print(f"{n} khong la so nguyen to")